MapboxNavigation/Error.swift

```diff
@@ -69,5 +68,12 @@ public enum SpeechError: LocalizedError {
      - parameter instruction: The instruction that failed.
      - parameter progress: the offending `RouteProgress` that omits the expected `SpeechLocale`.
      */
-    case undefinedSpeechLocale(instruction: SpokenInstruction, progress: RouteProgress)
+    case undefinedSpeechLocale(instruction: SpokenInstruction, progress: RouteProgress) // to be removed?
```

Comment:
Given the points about MapboxSpeechSynthesizer needing `speechLocale`, I’m not sure we can remove this case just yet.